### PR TITLE
Add support to show current date and time

### LINKED ISSUE #238: Add support to show current date and time
In $PS1 setting \d \t allows to print the current date and time.

### PR DESCRIPTION
**Pull request by flybyray**
Fixes #238

This allows to give a time layout as argument. See details [Time.Format](https://pkg.go.dev/time#Time.Format)

```shell
function _update_ps1() {
    PS1="$(powerline-go -error $? -jobs $(jobs -p | wc -l) -newline -time '06-01-02 15:04:05' -modules time,cwd,exit,git,hg,host,jobs,perms,root,ssh,user,venv )"
}
```

If no argument given to time option it will use the default before this pull request.

### CODE REVIEW

File: args.go [hunk 1]
```diff
@@ -188,6 +189,10 @@ var args = arguments{
 		"duration",
 		defaults.Duration,
 		comments("The elapsed clock-time of the previous command")),
+	Time: flag.String(
+		"time",
+		defaults.Time,
+		comments("The elapsed clock-time of the previous command")),
```
Comment (justjanne) [inline args.go:195]: Please correct the description of this flag

**Review by justjanne — CHANGES_REQUESTED** [review]

**Comment by embeddedc** [comment]
Could be merged now?

**Comment by vitaly-zdanevich** [comment]
And how to enable this?

**Comment by embeddedc** [comment]
> And how to enable this?

`-modules time`

**Pull request merged**